src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -347,6 +347,16 @@ void assertParent(Collection<Class<? extends Element>> permittedParents) throws
     }
   }
 
+  /**
+   * Check that the element's allowed parent is limited to one specific element type.
+   */
+  void assertSingleParent(Element permittedParent) throws InvalidInputException {
```

Comment:
Why do we need this method? Running `assertParent` with a `Collection.singleton` achieves the same thing. Multiple methods = more possibilities to have diverging code and less clarity to developers.

Reply:
Changed into FormElement class with Override of assertParent method